### DISCUSSION TITLE
[Docker] Official Docker image

### DISCUSSION THREAD

**Discussion by hacketiwack** [transmission/transmission#6071] [Ideas]
### What is the issue?

Not an issue, but more a suggestion.

Currently, there are options to deploy Transmission as a container - i.e. https://hub.docker.com/r/linuxserver/transmission.
However, it would be great to have a clean official Transmission Docker image with no additional packages (e.g., unrar, python, 7zip, etc.)

IMHO, this could increase security by attesting the source of the software, avoiding external packages that are not essential for the Transmission to run.

It would also make the releases proactive, by always having the latest version available.

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

N/A (latest)

**Comment by killemov** [comment]
The [Docker images by linuxserver.io](https://fleet.linuxserver.io/) are fine (foine) and up-to-date. They have cast a slightly wider net to accommodate scripting and such. And you could always roll your own.

**Comment by hacketiwack** [comment]
Actually, I'm not satisfied with the images provided by Linuxserver.io. Hence my request here.
They have a weird startup script that creates useless directories. To summarise, they over-engineered it.

**Comment by elisimpson** [comment]
If you want a simpler implementation I recommend gists:

https://hub.docker.com/r/gists/transmission

**Comment by fopina** [comment]
I'd like to second this: using a container image from the team writing the same software not only provides better maintenance (and updates) but also one less (weak) link in the supply chain.

Is there a reason not to have one, published directly to ghcr?

**Reply by DoctorD90** [reply]
This would be awesome! <3 There is also the [officlal GitHub guide ](https://docs.github.com/en/actions/tutorials/use-containerized-services/create-a-docker-container-action), and it could be based on the debian-latest to be sure of compatibility and max stability! Otherwise ubuntu too.
